salesforcecli/plugin-release-management
# Empty release flag

### What does this PR do?
Adds an `--empty` flag for creating an empty release PR. Useful for creating a PR to push changes to later.

QA: 
- Pull branch
- cd to your local `sfdx-cli`
- Run: `../plugin-release-management/bin/dev cli:release:build --start-from-npm-dist-tag latest-rc --release-channel latest-rc --patch --empty --build-only`
- Ensure the only change is made is the patch version in the `package.json`

### What issues does this PR fix or reference?
[@W-13082275@](https://gus.my.salesforce.com/apex/ADM_WorkLocator?bugorworknumber=W-13082275)
